### PR TITLE
Elaborate on how we version the documentation

### PR DESCRIPTION
**Pull request by artemkovalyov**
## What Has Changed?

As we discussed I added notes on documentation versioning.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)

### CODE REVIEW

File: docs/js/release-policy.mdx [hunk 1]
```diff
@@ -58,10 +60,48 @@ The solution is usually upgrading TypeScript to the latest stable version.
...
+:::tip
+
+We will always provide a branch with the latest snapshot of the documentation before the major version release.
+You can use that branch to host documentation for the earlier version locally.
+
+:::
+
+### Fixes and Maintenance
+
+- We maintain only the **latest major release** of the SAP Cloud SDK for JavaScript.
```
Comment (florian-richter) [inline docs/js/release-policy.mdx:104]:
```suggestion
- We only provide new features and fixes for the **latest major release** of the SAP Cloud SDK for JavaScript.
```

**Review by florian-richter — COMMENTED** [review]
Looks good with some small suggestions for improvement.

**Comment by artemkovalyov** [comment]
@florian-richter , give it another look.

File: docs/js/release-policy.mdx [hunk 2]
```diff
@@ -58,10 +60,48 @@ The solution is usually upgrading TypeScript to the latest stable version.
...
 - Following the upgrade steps should take less than a day of development effort regardless of the project size.
 
-### Fixes, Maintenance, Documentation
+### Documentation
 
-- We maintain only the **latest major release** of the SAP Cloud SDK for JavaScript.
 - The documentation is always up to date with the **latest major release**.
+- We will use tabs to demostrate a difference between the versions where required.
+
+<Tabs groupId="version" defaultValue="major" values={[ { label: 'v2', value: 'major' }, { label: 'v1', value: 'deprecated' } ]}>
```
Comment (marikaner) [inline docs/js/release-policy.mdx:68]: Can we use `v1` and `v2` as technical references and `"Version 1"` and `"Version 2"` as titles for the tabs? I think that would make it very clear.

**Review by marikaner — CHANGES_REQUESTED** [review]
LGTM, but I would like to suggest other titles for the tabs. I think major and deprecated could be hard to understand.

File: docs/js/release-policy.mdx (hunk 2 above)
Comment (artemkovalyov) [reply docs/js/release-policy.mdx:68]: As we always have just 2 versions I thought this will make it easier to maintain with keeping major a default.

Comment (artemkovalyov) [reply docs/js/release-policy.mdx:68]: Also with "Version 1", "Version 2" it's somewhat unclear about the version of what. I'd say we can use SDK 1.x, SDK 2.x or smth like this.

Comment (artemkovalyov) [reply docs/js/release-policy.mdx:68]:
```suggestion
<Tabs groupId="version" defaultValue="major" values={[ { label: 'SDK 2.x', value: 'major' }, { label: 'SDK 1.x', value: 'deprecated' } ]}>
```

**Review by marikaner — APPROVED** [review]

**Review by florian-richter — APPROVED** [review]

**Pull request merged**